src/main/scala/uk/gov/nationalarchives/tdr/api/http/ApiServer.scala

```diff
@@ -21,7 +22,7 @@ object ApiServer extends App {
 
   scala.sys.addShutdownHook(() -> shutdown())
 
-  val routes = new Routes(ConfigFactory.load())
+  val routes = new Routes(ConfigFactory.load(), DbConnection.db)
```

Comment:
This argument in the `Routes` class isn't being used so you can take it out.

Reply:
Ah, thanks. That was a mistake, the Routes class is supposed to use the db passed into it, but what is weird is that the `"The db" should "return 500 Internal Server Error if the db is down"` test was still working even though it closes the db and now works when I make the Routes class use the db that has been passed to it

Reply:
Ahhh, I know why. It's because that endpoint throws the same error type for two reasons: if there are no bodies in the DB AND if the db is down so it was just catching the former as it was essentially the same as the test above it (since it wasn't using the new closed test db that I added) 🤦‍♂️ . Good catch; thanks 👍